### DISCUSSION TITLE
unable to view archived classrooms

### DISCUSSION THREAD

**Discussion by mdilyasf** [community/community#57999] [GitHub Education]
### Hi everyone,

Hi, 

In github classroom, I am unable to change in the view options from "active" to "archive" although I have archived classrooms. May I know is this a bug?

**Comment by JohnMLarkin** [comment]
It is an awkward work around, but you can manually update the URL, changing the value after `view=` from `Active` to `Archived`.

https://classroom.github.com/classrooms?utf8=%E2%9C%93&query=&view=Archived&sort_by=Newest%20first
